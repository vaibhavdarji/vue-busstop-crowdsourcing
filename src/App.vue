<template>
  <v-app id="app">
    <app-bar></app-bar>

    <v-content transition="slide-x-transition">
      <router-view></router-view>
    </v-content>
  </v-app>
</template>

<script>
import AppBar from "@/components/AppBar";

export default {
  name: "App",
  components: {
    AppBar
  }
};
</script>

<style>
</style>
