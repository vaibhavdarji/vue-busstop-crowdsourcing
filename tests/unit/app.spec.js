import { shallowMount} from "@vue/test-utils";
import App from "@/App.vue";
import Vuetify from "vuetify";
import Vue from "vue";
import VueRouter from 'vue-router';
Vue.use(Vuetify);
Vue.use(VueRouter);

describe("App.vue", () => {
  it("should match snapshot", () => {
    const wrapper = shallowMount(App, {
      Vue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
