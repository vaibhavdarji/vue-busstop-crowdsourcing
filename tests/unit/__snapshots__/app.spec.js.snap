// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should match snapshot 1`] = `
<v-app-stub
  id="app"
>
  <app-bar-stub />
   
  <v-content-stub
    tag="main"
    transition="slide-x-transition"
  >
    <router-view-stub
      name="default"
    />
  </v-content-stub>
</v-app-stub>
`;
